TACC/Core-Portal
# task/FP-1102 - Add ability to compress folder to folder download message modal

## Overview: ##

Add the Compress form to the folder download message modal. 

## Related Jira tickets: ##

* [FP-1102](https://jira.tacc.utexas.edu/browse/FP-1102)

## Summary of Changes: ##
* Removed instructions on how to compress a folder from DataFilesDownLoadMessageModal.js
* Inserted compress form from DataFilesCompressModal.js in place of instructions into DatafilesDownloadMessageModal.js
* Updated DataFilesDownloadMessageModal.test.js (with a lot of help from Alex)

## Testing Steps: ##
1. Select a folder from the "Data Files" page in the portal.
2. Click on the "Compress" button in the Data Files toolbar and follow the instructions to compress.
3. Repeat step one, selecting the same folder selected in step one.
4. Click on the "Download" button in the Data Files toolbar and follow the instructions to compress.
5. Same results should come from both processes.

## UI Photos:
Download Modal without Compression Feature (Before Changes)
![Download without Compress Feature](https://user-images.githubusercontent.com/77903391/134575563-60f5fd87-ea06-47f9-bb98-81570751cce6.png)

Download Modal with Compression Feature (After Changes)
![Download with Compression Feature](https://user-images.githubusercontent.com/77903391/134575663-ff236400-7be9-4e6b-bc5f-ae6d23af85eb.png)

## Notes: ##
* Data Files "Compress" fails in development when creating a .zip file with a filename containing spaces.  Development uses a different version of zippy than production (frontera-zippy).  Since both the code I've updated for Download as well as the code for Compress (which I didn't touch) both fail, I think the failure is related to zippy.
